docs/spelling_wordlist.txt

```diff
@@ -9,14 +9,21 @@ CLI
 coopetition
 deployer
 github
+gitlab
+GitLab
+Gitlab
```

Comment:
we should use one spelling everywhere if possible

Reply:
Agreed, official one is GitLab.

Reply:
yes, but the docs may include lowercase `gitlab` somewhere as a part of a command/url etc. So the `spelling_wordlist` should include those possibilities (or cause annoying failing builds).

Reply:
those are fine i guess